app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,9 @@
     <%= csp_meta_tag %>
     <%= stylesheet_pack_tag 'application', media: 'all' %>
     <%= javascript_pack_tag 'application' %>
+  <%=javascript_include_tag "https://www.authy.com/form.authy.min.js" %>
```

Comment:
Is it possible to host these locally? Are they available in an npm package?

Reply:
Yes there absolutely is an npm package. Just added it and removed the external references.